Packs/DSPM/Integrations/DSPM/DSPM.py

```diff
@@ -0,0 +1,903 @@
+GET_DATA_TYPE_FINDINGS_ENDPOINT: str = "/v1/data-type-findings"
+GET_ALERTS_LIST: str = "/v1/alerts"
+GET_RISK_FINDING_BY_ID: str = "/v1/risk-findings/id/"
+INCIDENT_STATUS = {"OPEN": 1, "INVESTIGATING": 2, "HANDLED": 2, "CLOSED": 2}
+RISK_STATUS = {"Active": "OPEN", "Closed": "INVESTIGATING", "Pending": "INVESTIGATING"}
+
+SUPPORTED_CLOUD_PROVIDERS = ["AWS", "AZURE", "GCP", "SNOWFLAKE", "FILE_SHARE", "O365"]
+SUPPORTED_CATEGORIES = [
+    "SECURITY",
+    "COMPLIANCE",
+    "GOVERNANCE",
+    "SECURITY_AND_COMPLIANCE",
+    "SECURITY_AND_GOVERNANCE",
+    "COMPLIANCE_AND_GOVERNANCE",
+    "SECURITY_AND_COMPLIANCE_AND_GOVERNANCE",
+]
+SUPPORTED_STATUSES = [
+    "OPEN",
+    "CLOSED",
+    "UNIMPORTANT",
+    "WRONG",
+    "HANDLED",
+    "INVESTIGATING",
+]
+SUPPORTED_SERVICE_TYPES = [
+    "ATHENA",
+    "AURORA",
+    "AWS_BACKUP",
+    "DOCUMENTDB",
+    "DYNAMODB",
+    "DAX",
+    "EMR",
+    "EBS",
+    "EFS",
+    "ELASTICACHE",
+    "FSX",
+    "KINESIS_DELIVERY_STREAM",
+    "MEMORYDB",
+    "NEPTUNE",
+    "QLDB",
+    "RDS",
+    "REDSHIFT",
+    "S3",
+    "TIMESTREAM",
+    "OPENSEARCH",
+    "COSMOS_DB",
+    "SYNAPSE",
+    "CACHE_FOR_REDIS",
+    "MARIA_DB",
+    "MYSQL_SERVER",
+    "POSTGRESQL_SERVER",
+    "SQL_SERVER",
+    "STORAGE_ACCOUNT",
+    "SQL_MANAGED_INSTANCE",
+    "ANF",
+    "CLOUD_STORAGE",
+    "CLOUD_SQL",
+    "BIG_QUERY",
+    "FILE_STORE",
+    "CLOUD_SPANNER",
+    "MEMORY_STORE",
+    "BIG_TABLE",
+    "FIRE_STORE",
+    "UNMANAGED_AWS_MYSQL",
+    "UNMANAGED_AWS_ORACLE_SERVER",
+    "UNMANAGED_AWS_MONGO_DB",
+    "UNMANAGED_AWS_POSTGRESQL",
+    "UNMANAGED_AWS_REDIS",
+    "UNMANAGED_AWS_SQLITE",
+    "UNMANAGED_AWS_MS_SQL",
+    "UNMANAGED_AWS_MARIA_DB",
+    "UNMANAGED_AWS_NEO4_J",
+    "UNMANAGED_AWS_ELASTIC",
+    "UNMANAGED_AWS_COCKROACH_DB",
+    "UNMANAGED_AWS_AEROSPIKE",
+    "UNMANAGED_AWS_SCYLLA_DB",
+    "UNMANAGED_AZURE_MYSQL",
+    "UNMANAGED_AZURE_ORACLE_SERVER",
+    "UNMANAGED_AZURE_MONGO_DB",
+    "UNMANAGED_AZURE_POSTGRESQL",
+    "UNMANAGED_AZURE_REDIS",
+    "UNMANAGED_AZURE_SQLITE",
+    "UNMANAGED_AZURE_MS_SQL",
+    "UNMANAGED_AZURE_MARIA_DB",
+    "UNMANAGED_AZURE_NEO4_J",
+    "UNMANAGED_AZURE_ELASTIC",
+    "UNMANAGED_AZURE_COCKROACH_DB",
+    "UNMANAGED_AZURE_AEROSPIKE",
+    "UNMANAGED_AZURE_SCYLLA_DB",
+    "UNMANAGED_GCP_MYSQL",
+    "UNMANAGED_GCP_ORACLE_SERVER",
+    "UNMANAGED_GCP_MONGO_DB",
+    "UNMANAGED_GCP_POSTGRESQL",
+    "UNMANAGED_GCP_REDIS",
+    "UNMANAGED_GCP_SQLITE",
+    "UNMANAGED_GCP_MS_SQL",
+    "UNMANAGED_GCP_MARIA_DB",
+    "UNMANAGED_GCP_NEO4_J",
+    "UNMANAGED_GCP_ELASTIC",
+    "UNMANAGED_GCP_COCKROACH_DB",
+    "UNMANAGED_GCP_AEROSPIKE",
+    "UNMANAGED_GCP_SCYLLA_DB",
+    "SNOWFLAKE_DB",
+    "FILE_SHARE",
+    "ONE_DRIVE",
+    "SHARE_POINT",
+    "AZURE_OPEN_AI_DEPLOYMENT",
+    "VERTEX_ENDPOINT",
+]
+SUPPORTED_LIFECYCLE = ["RUNNING", "STOPPED", "DELETED"]
+SORTING_ORDER = ["ASC", "DESC"]
```

Comment:
please remove **SORTING_ORDER** you dont use it in the code, same for **RISK_STATUS**,**INCIDENT_STATUS** and the **map_status**, **severity_to_dbot_score** methods
```suggestion
```

Reply:
Sure.